### ISSUE TITLE
Can nemo work on my custom dataset?

### ISSUE THREAD

**Issue by EninumXJ** [wangkua1/nemo-cvpr2023#2]
Now i have some multiview videos of the same action in the wild, how can I render the pose using Nemo?

**Comment by wangkua1** [comment]
Yes! We're currently working on a tutorial for running NeMo on your own videos.  Stay tuned!

**Comment by wangkua1** [comment]
Hi, you can find an example now in the custom_video directory!
